float64[] pose
bool reset
bool align
---
bool success
